script.js

```diff
@@ -0,0 +1,49 @@
+let startBtn = document.querySelector("#startButton");
+startBtn.addEventListener("click", playGame);
+//ash says we can use this as a global object
+let gameState = ["", "", "", "", "", "", "", "", ""];
+
+function decideTurn(currentPlayer) {
+    if (currentPlayer === 'o') {
+        return 'x';
+    }
+    return 'o';
+}
+function addToScreen(tile, currentPlayer) {
+    tile.innerHTML = currentPlayer;
+}
+
+function alterGameState(gameState, clicked, currentPlayer) {
+    if(gameState[clicked] === "") {
+        gameState[clicked] = currentPlayer;
+        return true;
+     }
+    return false;
+}
+
+function playGame() {
+    // this determines turn order
+    // this variable holds alternating player symbols
+    let currentPlayer = 'x';
+    const tiles = document.querySelectorAll('.tile');
+    //this is a boolean to regulate whether a player can click a tile.
+    let notYetClicked;
```

Comment:
Now that you're defining notYetClicked in the loop, it doesn't need to be here at all.